identity-service/src/notifications/sendNotifications/index.js

```diff
@@ -1,7 +1,7 @@
 const models = require('../../models')
 const { notificationTypes } = require('../constants')
 const { fetchNotificationMetadata } = require('../fetchNotificationMetadata')
-const formatNotification = require('./formatNotification')
+const formatNotifications = require('./formatNotification')
```

Comment:
how did this work before?

Reply:
Since it's a top level export, you can call it whatever you want!